modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,77 @@ This release specifically highlights agility, performance, and manageability and
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-552]
+== Release 5.5.2
+Couchbase Server 5.5.2, released in October 2018, is the second maintenance release in the 5.5.x series for Couchbase Server.
+This maintenance release includes fixes to the index and query services, XDCR, and installer components.
+
+[#fixed-issues-552]
+*Fixed Issues*
+
+*Cross Datacenter Replication(XDCR)*
+
+[#table_fixedissues_xdcr_552,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31141[MB-31141^]
+| Fixed an issue where replication from version 4.5.1 to version 5.5.x caused corruption on deleted documents leading to inability to rebalance.
+|===
+
+*Index Service*
+
+[#table_fixedissues_data_552,cols="25,66"]
```

Comment:
Should this be #table_fixedissues_index_552, not #table_fixedissues_data_552?

Reply:
Fixed